### DISCUSSION TITLE
Minimize

### DISCUSSION THREAD

**Discussion by Corfucinas** [pradyunsg/furo#368] [Ideas] [answered]
Is it possible to produce a `minified` HTML and CSS?

**Comment by pradyunsg** [comment, marked as answer]
I don't believe Sphinx provides any straightforward mechanisms to do that -- Furo's CSS and JS is minified, and the HTML is not too bloated either.

**Reply by Corfucinas** [reply]
Thanks for reply, I'll take it up at Sphinx
